packages/server/src/fhir/repo.test.ts

```diff
@@ -241,6 +241,63 @@ describe('FHIR Repo', () => {
       expect(patient2.name?.[1]?.family).toEqual('Jones');
     }));
 
+  test('Patch patient', () =>
+    withTestContext(async () => {
+      const patient = await systemRepo.createResource<Patient>({
+        resourceType: 'Patient',
+        name: [{ given: ['Suzy'], family: 'Smith' }],
+      });
+      const result = await systemRepo.patchResource('Patient', patient.id as string, [
+        {
+          op: 'add',
+          path: '/active',
+          value: true,
+        },
+      ]);
+
+      expect(result.id).toEqual(patient.id);
```

Comment:
I added this spec to wrap my head around the `patchResorce` function - I copied the spec from throw-error-on-patch-issue-3888. I'm a little unclear as I thought the patch would modify the resource in place, instead of having to use and check the `result` object